.rubocop.yml

```diff
@@ -111,7 +111,7 @@ Metrics/MethodLength:
     - "spec/**/*"
 
 Metrics/ModuleLength:
-  Max: 200
+  Max: 210
```

Comment:
I don't see any modules increasing in length...

Reply:
`spec/support/vacancy_helpers.rb` is now 202 lines of code